ibuffer-projectile.el

```diff
@@ -98,10 +98,16 @@ If the file is not in a project, then nil is returned instead."
 (defun ibuffer-projectile-generate-filter-groups ()
   "Create a set of ibuffer filter groups based on the projectile root dirs of buffers."
   (let ((roots (ibuffer-remove-duplicates
-                (delq nil (mapcar 'ibuffer-projectile-root (buffer-list))))))
+                (delq nil (mapcar (lambda (buf)
+                                    (let ((root (ibuffer-projectile-root buf)))
+                                      (when root
+                                        (cons (with-current-buffer buf
```

Comment:
Oh I did not realize that each new commit to that branch in my fork would pile up in the pull request.

Anyway, I pushed a new commit 6a0530b following your comment and tried to mimic `ibuffer-vc-root`. Hopefully, it should address your concern.

If you want me to rebase these commits for a cleaner revision history, let me know.